test/integration/framework/src/siftool/test_utils.py

```diff
@@ -347,12 +355,14 @@ def smart_contract_get_past_events(self, sc, event_name, from_block=None, to_blo
         finally:
             self.w3_conn.eth.uninstall_filter(filter.filter_id)
 
-    def tx_deploy_new_generic_erc20_token(self, deployer_addr, name, symbol, decimals):
+    def tx_deploy_new_generic_erc20_token(self, deployer_addr, name, symbol, decimals, cosmosDenom=None):
         # return self.tx_deploy("SifchainTestToken", self.operator, [name, symbol, decimals])
         if on_peggy2_branch:
             # Use BridgeToken
             assert self.generic_erc20_contract == "BridgeToken"
-            cosmosDenom = "erc20denom"  # TODO Dummy variable since we're using BridgeToken instead of SifchainTestToken
+            if cosmosDenom is None:
```

Comment:
for ibc token and double peg token, we set the cosmosDenom. It doesn't make sense to set cosmosDenom for ERC20 in the original network. And we can't get it since the contract not created yet.